identity-mdoc/src/test/java/com/android/identity/mdoc/response/DeviceResponseGeneratorTest.kt

```diff
@@ -544,6 +545,16 @@ class DeviceResponseGeneratorTest {
         Assert.assertEquals("foo1", doc.getIssuerEntryString("ns2", "bar1"))
     }
 
+    @Test
+    @Throws(Exception::class)
+    fun testDocumentGeneratorNoDocuments() {
+        val deviceResponseGenerator = DeviceResponseGenerator(20)
+        val encodedDeviceResponse = deviceResponseGenerator.generate()
+
+        Assert.assertEquals(TestVectors.DEVICE_RESPONSE_NO_DOCUMENTS_STATUS_20.fromHex, encodedDeviceResponse)
```

Comment:
Generally stuff in `TestVectors.kt` are the test vectors from Annex D. I think it would be cleaner to do

```
 Assert.assertEquals("<insert pretty-printed-CBOR>", Cbor.toDiagnostics(encodedDeviceResponse, setOf(DiagnosticOption.PRETTY_PRINT)))
```

and then maybe add a comment above to say that there's no "documents" map entry.